monai/transforms/transforms.py

```diff
@@ -24,6 +25,65 @@
 export = monai.utils.export("monai.transforms")
 
 
+@export
+class LoadNifti:
```

Comment:
could we also have an additional `LoadNiftiMultichannel` so that we can stack multiple files at the first dim (channel dim)?

Reply:
Hi @wyli ,

I think that should be another transform: "MergeToChannels", we have it in Clara.
Thanks.